src/loaders/lottie/tvgLottieModel.h

```diff
@@ -184,8 +184,12 @@ struct LottieTextRange
     Based based = Chars;
     Shape shape = Square;
     Unit rangeUnit = Percent;
+    uint8_t seed = 0;
```

Comment:
Logically it's not a seed, just a random value.

Suggestion:  
```
uint8_t random = 0;  // allocate random = rand(sizeof(uint8_t)) in Parsing "rd"
bool randomize = false;  <--- remove
```
You can apply the random when it's value is not zero.